packages/react-components/react-components/src/index.ts

```diff
@@ -634,4 +634,10 @@ export type {
   TooltipTriggerProps,
 } from '@fluentui/react-tooltip';
 
-export type { PositioningProps, PositioningShorthand, PositioningShorthandValue } from '@fluentui/react-positioning';
+export type {
+  PositioningProps,
```

Comment:
Do you also want to export the types used within these types such as Offset, Alignment, Position?